src/bindings/jna/libelektra5j/pom.xml.in

```diff
@@ -74,6 +74,7 @@
 							</systemPropertyVariables>
 							<excludes>
 								<exclude>**/KDBTest.java</exclude>
+								<exclude>**/GOptsTest.java</exclude>
```

Comment:
Please write in doc/todo/TESTING about these disabled tests with a reason why they were disabled.

Reply:
I don't actually know why exactly the KDB tests don't work. I just saw this

https://github.com/ElektraInitiative/libelektra/blob/4c45251d40a47723cf61381f77aeca31ad90ce43/src/bindings/jna/CMakeLists.txt#L117-L125

and searched for `env-test-without-kdb` to find where I had to add `GOptsTest` to exclude it (since it uses KDB).